dask_cuda/explicit_comms/dataframe/shuffle.py

```diff
@@ -542,7 +543,22 @@ def shuffle(
     return ret
 
 
-def get_rearrange_by_column_tasks_wrapper(func):
+def _use_explicit_comms() -> bool:
+    """Is explicit-comms and available?"""
+    if dask.config.get("explicit-comms", False):
+        try:
+            import distributed.worker
```

Comment:
Will this ever fail? I think we could potentially just import this at the top and avoid the `except ImportError` altogether, or am I overlooking something?

Reply:
`distributed.worker.get_client()` will fail if running locally with a client

Reply:
I guess, we can move `import distributed.worker` to the top though

Reply:
Will the `import distributed.worker` fail as well or just `distributed.worker.get_client()`?

Reply:
Just noticed, that we import `from distributed.worker import Worker` at the top already, so I hope not :)

Reply:
fixed